### PR TITLE
Register lumix.is-a.dev

### PR DESCRIPTION
**Pull request by nyxaroo**
Register lumix.is-a.dev with CNAME record pointing to o-lumix.github.io.

### CODE REVIEW

File: domains/lumix.json
Comment (CuteDog5695) [inline domains/lumix.json:9]: 404 page, but if you have the website hosted but not on the root domain then send the website link to the page

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/lumix.json
Comment (o-lumix) [reply domains/lumix.json:9]: Now is good?

**Review by iostpa — APPROVED** [review]
seems good to me now, but you'd have to wait for the trial maintainer to merge it since willy screwed up codeowner file so that us maintainers cant merge PRs

**Pull request merged**
